packages/front-end/services/UserContext.tsx

```diff
@@ -222,7 +223,7 @@ export function UserContextProvider({ children }: { children: ReactNode }) {
   // Refresh organization data when switching orgs
   useEffect(() => {
     if (orgId) {
-      refreshOrganization();
+      void refreshOrganization();
```

Comment:
Curious why `void` is necessary here?

Reply:
it's not. It just signals that we intentionally ignore the returned promise, rather than forgot to handle (await) it.